src/fbs/header.fbs

```diff
@@ -52,6 +60,7 @@ table Crs {
   name: string;                 // Human readable name of this SRS
   description: string;          // Human readable description of this SRS
   wkt: string;                  // Well-known Text Representation of the Spatial Reference System
+  code_string: string;          // Text ID of the Spatial Reference System assigned by the organization (rarely used)
```

Comment:
not clear what that would be. Can you give an example ?

Reply:
This is to resolve https://github.com/bjornharrtell/flatgeobuf/issues/44, I think you know more about it than me? 🙂

Reply:
oh !!! my memory is so terrible.  Perhaps:
```suggestion
  code_string: string;          // ID of the Spatial Reference System assigned by the organization, when it is not an integer, and thus cannot be set into code.
```

Reply:
I did understand it as text part only (e.g. `EPSG`) used together with the numerical code (e.g. `4326`), but in @rouault's example `IGNF:RGF93` there is no numerical part. Maybe it should be documented, which one is used when both the numerical srid and code_string is set.

Reply:
I've attempted to clarify. I've kept the word rare in the sentence to indicate that it's not to be preferred instead of `code` when possible.